api/capz/v1alpha3/azure_cluster_types.go

```diff
@@ -0,0 +1,25 @@
+package v1alpha3
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:object:root=true
+type AzureCluster struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+
+	Spec AzureClusterSpec `json:"spec"`
+}
```

Comment:
These types were added for Azure CNI support IIRC. Will they eventually move to another repo?

Reply:
I'm doing a trick here to avoid importing CAPZ. Described in more detail here https://github.com/giantswarm/cluster-apps-operator/pull/118#discussion_r755552801

Reply:
Cool, good to keep the imports as simple as we can.